### ISSUE TITLE
Scans broken when running from alternate metadata directory

### ISSUE THREAD

**Issue by bengalih** [jellyfin/jellyfin#3796]
When running scans with an alternatively configured metadata/cache directory, the scans fail with nothing generated in the logs:

**System (please complete the following information):**
 - OS: Windows
 - Virtualization:none
 - Clients: Browser
 - Browser: Chrome 84.0.4147.89 
 - Jellyfin Version: 10.6.1
 - Playback: n/a
 - Installed Plugins: Pushbullet
 - Reverse Proxy: NGINX
 - Base URL: no
 - Networking: n/a
 - Storage: local

I've got my jellyfin data stored in the following location while testing multiple versions:

`D:\ProgramData\jellyfin\data.10.6.1\metadata`

As per issue #3750 the command line does not honor setting cachedir (or metadata), so I have modified the following in the system.xml to properly point:

```
<CachePath>D:\ProgramData\jellyfin\data10.6.1\cache</CachePath>
<MetadataPath>D:\ProgramData\jellyfin\data.10.6.1\metadata</MetadataPath>
```

Dashboard info properly reports the directories loaded from the specified location, but a scan of a library fails.
All that happens is the WebUI appears to reload (JF logo), and absolutely nothing of relevance is posted in the logs.
With Debug logging enabled all I see when attempting to refresh is:

```
[21:26:21] [DBG] [14] Emby.Server.Implementations.Session.WebSocketController: Removing websocket from session a0afef8a998d306ee589e5cc842f1f37
[21:26:21] [DBG] [14] Emby.Server.Implementations.Session.SessionManager: Disposing session controller WebSocketController
[21:26:21] [DBG] [14] Emby.Server.Implementations.Session.SessionWebSocketListener: WebSocket Emby.Server.Implementations.HttpServer.WebSocketConnection is closed.
[21:26:21] [INF] [14] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS 10.10.10.1 closed
[21:26:23] [DBG] [14] Emby.Server.Implementations.HttpServer.HttpResultFactory: Transmit file D:\Program Files\jellyfin\system.10.6.1\jellyfin-web\config.json
[21:26:24] [DBG] [17] Emby.Server.Implementations.HttpServer.HttpResultFactory: Transmit file D:\Program Files\jellyfin\system.10.6.1\jellyfin-web\strings\en-us.json
[21:26:28] [INF] [23] Emby.Server.Implementations.HttpServer.HttpListenerHost: WS 10.10.10.1 request
[21:26:28] [DBG] [23] Emby.Server.Implementations.Session.SessionManager: Creating new WebSocketController
[21:26:28] [DBG] [23] Emby.Server.Implementations.Session.WebSocketController: Adding websocket to session a0afef8a998d306ee589e5cc842f1f37
[21:26:28] [DBG] [23] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS 10.10.10.1 received message: {"MessageType": "KeepAlive", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": null, "$type": "WebSocketMessage`1"}
[21:26:29] [DBG] [9] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS 10.10.10.1 received message: {"MessageType": "ScheduledTasksInfoStart", "MessageId": "00000000-0000-0000-0000-000000000000", "ServerId": null, "Data": {"ValueKind": "String", "$type": "JsonElement"}, "$type": "WebSocketMessage`1"}
[21:26:29] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Getting topLibraryFolders
[21:26:29] [DBG] [23] Emby.Server.Implementations.Library.LibraryManager: Getting refreshQueue
[21:26:29] [DBG] [9] MediaBrowser.Api.ScheduledTasks.ScheduledTasksWebSocketListener: WS 10.10.10.1 begin transmitting to ScheduledTasksWebSocketListener
[21:26:31] [DBG] [11] Emby.Server.Implementations.Session.SessionWebSocketListener: Watching 1 WebSockets.
```

Moving /cache and /metadata back to just "/data" while loading the rest from the /data.10.6.1 directory also seems to fail in the same way.

In short, it appears running it from anywhere other than /data location seems to fail.
